### PR TITLE
Cleaned cabal files

### PR DESCRIPTION
**Pull request by coot**
- Updated cabal files
- Cleaned cabal files
- Update copyright
- Consistent indents in cabal files
- Consistent license-files declarations in cabal files
- Update all cabal files to version 3.0
- Add project-config to all cabal files

### CODE REVIEW

**Review by deepfire — APPROVED** [review]
Thank you @coot !

**Review by Jimbo4350 — DISMISSED** [review]

File: bench/locli/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2021 Input Output (Hong Kong) Ltd.
```
[the remark this answers is not in the document]
Comment (newhoggy) [reply bench/locli/LICENSE:190]: Should the copyright say IOG now?  Or do we stick with IOHK?

File: cardano-testnet/cardano-testnet.cabal [hunk 2]
```diff
@@ -10,11 +10,10 @@ license-files:          LICENSE
                         NOTICE
 build-type:             Simple
 
-common base                         { build-depends: base                             >= 4.14       && < 4.15     }
-
 common project-config
   default-language:     Haskell2010
   default-extensions:   NoImplicitPrelude
+  build-depends: base >= 4.14 && < 4.15
```
Comment (newhoggy) [inline cardano-testnet/cardano-testnet.cabal:16]: Indent please.

**Comment by newhoggy** [comment]
Do we know if all of our projects have been upgraded to use `aeson-2`?

If so we should stick a lower bound on that package everywhere.

**Review by newhoggy — APPROVED** [review]

**Pull request merged**